src/core/abr/representation_chooser.ts

```diff
@@ -154,8 +153,7 @@ function getConcernedRequest(
       duration: chunkDuration,
     } = request;
 
-    // TODO review this
-    if (Math.abs(segmentPosition - chunkTime) < chunkDuration) {
+    if ((chunkTime - segmentPosition) < chunkDuration) {
```

Comment:
If I get this straight:
``chunkTime`` is the starting time of the chunk
``segmentPosition`` is the soonest position in time where we need a segment
 
This function ``getConcernedRequest`` should return the request for the aforementioned segment.

Technically here, segmentPosition and chunkTime should be roughly equivalents.

From the wording of the function and JSDoc, it seems that ``segmentPosition`` can be anywhere in that segment.

Which means:

  - either we states cleary that ``segmentPosition`` should be close to the segment beginning and we check if ``segmentPosition`` and ``chunkTime`` are close enough (where we should define enough

  - either we allow ``segmentPosition`` to be anywhere in the segment, and we change the way we call this function and this condition (where it would be something like ``segmentPosition >= chunkTime && segmentPosition < (chunkTime + chunkDuration)``

In both of those cases, this condition is kind of off.